Automattic/wp-calypso
# Jetpack Plans: add new features to features table

This PR adds the two new jetpack plans features to the features table:

![image](https://cloud.githubusercontent.com/assets/1554855/21264356/28f41788-c39c-11e6-82b4-a4dca26c2040.png)

![image](https://cloud.githubusercontent.com/assets/1554855/21264467/ba27f4e0-c39c-11e6-8381-5a9494fdeaf8.png)

How to test
========
1. Go to https://calypso.live/plans/?branch=add/jeptack-plans-new-features and select a jetpack site
2. Go to http://calypso.live/jetpack/connect/vaultpress?branch=add/jeptack-plans-new-features
3. In both places, you should see videopress support & "monetize your site" as features for the business and pro plans, both in the yearly and in the monthly versions
